satpy/readers/nc_nwcsaf.py

```diff
@@ -162,14 +162,17 @@ def scale_dataset(self, dsid, variable, info):
         except AttributeError:
             pass
 
+        if 'palette_meanings' in variable.attrs:
+            variable.attrs['palette_meanings'] = [int(val)
```

Comment:
These are NetCDF4 files right? Can't you store a series of numbers in an attribute? I know this is just the reader of these files and not the writer, just curious.

Reply:
Yes it's nc files, but I'm not sure attributes can store anything else than atomic types (numbers or strings)